tox.ini

```diff
@@ -23,6 +23,8 @@ deps =
 commands =
   # safety measure to assure we do not accidentally run tests with broken dependencies
   {envpython} -m pip check
+  # Ensure that our fixtures are kept updated
+  {envpython} tools/regenerate-fixtures.py
```

Comment:
Doing this will mean that people that use `tox` would need to have `prettier` on their PATH (vs only installed by pre-commit)

Reply:
Yep. Precommit does not allow us to create a hook that uses two languages (node and python) so we need to take care of installing prettier ourselves.

Reply:
I would rather not force people to install npm + prettier for most local development, but let it run in CI to catch issues when people don't have it installed locally.

Reply:
(pre-commit hides the fact that it's getting installed)